src/services/api.js

```diff
@@ -0,0 +1,9 @@
+import axios from "axios";
+
+const api = axios.create({
+    baseURL:"https://cdn.contentful.com//spaces/nbe8tmkxy59u/environments/master/entries?access_token=ynfiaMt1N5Skp8U-uq7Z49Xc9OSl6g-je6-cEcapCqU&content_type=exhibition"
```

Comment:
What's this URL for?

Reply:
I copied the code from a previous project I did. I have changed it to the pets API